### ISSUE TITLE
The <template> element is deprecated. 

### ISSUE THREAD

**Issue by PikaChokeMe** [valor-software/ngx-bootstrap#1945]
```
The <template> element is deprecated. Use <ng-template> instead ("
[WARNING ->]<template [ngIf]="!isClosed"><div [class]="'alert alert-' + type" role="alert" [ngClass]="classes""): ng:///OMITTED/node_modules/ngx-bootstrap/alert/alert.component.d.ts.AlertComponent.html@1:0
```

I've noticed this error when upgrading to Angular v4 and compiling. I don't know if it has been fixed (I believe I'm on the latest version) and I'm sorry if it's a duplicate issue (I searched but didn't see anything).

This should be an easy to implement fix and I thought you may like to know. I hope this helps.

**Comment by fchiumeo** [comment]
https://github.com/valor-software/ngx-bootstrap/search?utf8=%E2%9C%93&q=ng-template&type=Issues

**Comment by Martin-Luft** [comment]
For Angular 4 just use the @next version which fixes this warning. This cannot be fixed in 1.6.X because it would break Angular 2 support. You can use "ngx-bootstrap": "2.0.0-beta.2" or npm install ngx-bootstrap@next
